doc/source/user_manual/rule_language.rst

```diff
@@ -961,6 +961,95 @@ In the following example two files are being used, but only the first existing f
         - PATH_TO_FILE_WITH_LIST
     description: '...'
 
+Selective Extractor
+===================
+
+The selective extractor requires the additional field :code:`selective_extractor`.
+The field :code:`selective_extractor.extract` has to be defined.
+It contains a dictionary of field names and values that should be extracted.
+If dot notation is being used, then all fields on the path are being automatically created.
+
+In the following example, the field :code:`extract_test.field.extract` with the value :code:`extracted value` is being extracted
```

Comment:
Is the field in the filter automatically the root of the extraction? If not I think it should be like this:

```suggestion
In the following example, the field :code:`field.extract` with the value :code:`extracted value` is being extracted
```